.github/workflows/ci.yml

```diff
@@ -71,14 +39,7 @@ jobs:
             ~/.cargo/git/db/
             target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
-      - name: Setup Rust
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          override: true
-          profile: minimal
-          target: ${{ matrix.config.target }}
       - uses: actions-rs/cargo@v1
         with:
-          command: build
-          args: --locked --target ${{ matrix.config.target }}
+          command: clippy
+          args: -- -W clippy::correctness -D warnings
```

Comment:
I think we still want to run the clippy check as a matrix on different platforms to capture differences in the build path on different platforms.

It may not be necessary for different desktop platforms, though. Maybe just web and native. It depends on whether or not we end up adding any OS specific doe into Punchy itself. I think since Bevy handles the abstraction over desktop platforms, we'd be good just checking Linux and web.

Reply:
Oh, and maybe we should add with and without the `--features debug` flag to the matrix, too. @odecay had mentioned the debug build kept failing because we don't tend to test it out as much.

Reply:
I think we could make those changes in a separate PR? So that we have more time to think them through.

Reply:
Fair enough.